optin/reporting/test/mock/current.version

```diff
@@ -0,0 +1,2 @@
+_ykRdiKlyoCZJAi4FPnptUcNL5FeNW9G
```

Comment:
can you rename mock folder to resources as it's more consistent with naming elsewhere